### PR TITLE
PCHR-3858: Add angulartics to reqangular

### PR DESCRIPTION
**Pull request by AkA84**
This PR simply adds the [angulartics](https://github.com/angulartics/angulartics) (v1.6.0) and [angulartics-google-tag-manager](https://github.com/angulartics/angulartics-google-tag-manager/)  (v0.0.2) modules to the `org.civicrm.reqangular` extension

The modules will be used in a later PR to add google analytics tracking to page views and user-triggered events

### CODE REVIEW

**Comment by igorpavlov** [comment]
@AkA84 if we have a reference or two words on *why* this package was added, it would be awesome.

File: org.civicrm.reqangular/src/common/vendor/angular/angulartics.min.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Angulartics
+ * (c) 2013 Luis Farzati http://angulartics.github.io/
+ * License: MIT
```
Comment (igorpavlov) [inline org.civicrm.reqangular/src/common/vendor/angular/angulartics.min.js]: Can we specify which version it is?

File: org.civicrm.reqangular/src/common/vendor/angular/angulartics-google-tag-manager.min.js [hunk 2]
```diff
@@ -0,0 +1 @@
+!function(angular){"use strict";angular.module("angulartics.google.tagmanager",["angulartics"]).config(["$analyticsProvider",function($analyticsProvider){function eventTrack(action,properties){var dataLayer=window.dataLayer=window.dataLayer||[];properties=properties||{},dataLayer.push({event:properties.event||"interaction",target:properties.category,action:action,"target-properties":properties.label,value:properties.value,"interaction-type":properties.noninteraction,userId:$analyticsProvider.settings.ga.userId})}$analyticsProvider.settings.ga={userId:null},$analyticsProvider.registerPageTrack(function(path){var dataLayer=window.dataLayer=window.dataLayer||[];dataLayer.push({event:"content-view","content-name":path,userId:$analyticsProvider.settings.ga.userId})}),$analyticsProvider.registerEventTrack(eventTrack),$analyticsProvider.registerExceptionTrack(function(properties){return properties.description?void eventTrack(properties.description,{category:"Exceptions",label:properties.stack}):void console.log("Property description is required to be set")}),$analyticsProvider.registerSetUsername(function(userId){$analyticsProvider.settings.ga.userId=userId})}])}(angular);
```
Comment (igorpavlov) [inline org.civicrm.reqangular/src/common/vendor/angular/angulartics-google-tag-manager.min.js]: This file does not contain empty last line.

Comment (igorpavlov) [inline org.civicrm.reqangular/src/common/vendor/angular/angulartics-google-tag-manager.min.js]: Can we include the version of the lib?

File: org.civicrm.reqangular/src/common/vendor/angular/angulartics.min.js (hunk 1 above)
Comment (AkA84) [reply org.civicrm.reqangular/src/common/vendor/angular/angulartics.min.js]: it's a vendor file, i'm using it as-is

File: org.civicrm.reqangular/src/common/vendor/angular/angulartics-google-tag-manager.min.js (hunk 2 above)
Comment (AkA84) [reply org.civicrm.reqangular/src/common/vendor/angular/angulartics-google-tag-manager.min.js]: it's a vendor file, i'm using it as-is

**Pull request merged**